index.js

```diff
@@ -47,7 +47,7 @@ module.exports = function(babel) {
     const stylesArguments = [];
     let classNameAttr = null;
     let originalCssValue;
-    const newAttrs = jsxAttrs.filter(attr => {
+    const transformedJsxAttrs = jsxAttrs.filter(attr => {
```

Comment:
Sorry to be a pain! But `transformedJsxAttrs` => transformed from what? A little 

```
/**
  * this does this
  */
```

would be helpful

Reply:
Then we can merge it 🚀